JS-test.js

```diff
@@ -0,0 +1,48 @@
+// 1 Programming Basics
+
+let x
+for (x = 10; x < 40; x++) {
+  if (x % 5 == 0 && x % 3 == 0) {
+    console.log(x + ' Jackpot!')
+  } else if (x % 3 == 0) {
+    console.log(x + ' This is divisible by 3')
+  } else if (x % 5 == 0) {
+    console.log(x + ' This is divisible by 5')
+  } else {
+    console.log(x)
+  }
+}
```

Comment:
Very nicely done! The format of both the code and the output is on point. Pay attention though, you go up 39, but do not reach 40. You should have used `<=`. 😉